lib/test_helpers.ml

```diff
@@ -0,0 +1,27 @@
+open Parsetree
+open Typedtree
+open Types
+
+(* Helper library to parse .mli content and return Types.signature *)
+let compile_interface (content : string) : Types.signature =
+  (* Parses and tokenizes the contents of the .mli file*)
+  let parse_interface content =
```

Comment:
Defining a local function here is a bit convoluted given it's just two lines of code.

I would either take it out and just define a `parse_interface` at the toplevel of the module to be used here or simply remove it.